src/Makefile.am

```diff
@@ -207,7 +206,6 @@ EXTRA_DIST += harfbuzz-icu.pc.in
 
 if HAVE_GOBJECT
 lib_LTLIBRARIES += libharfbuzz-gobject.la
-libharfbuzz_gobject_la_LINK = $(libharfbuzz_la_LINK)
```

Comment:
I added this but am worried that it makes more issue than solve. Perhaps CI bots can show whether it is useful a change or better to reverted to its previous state.

Reply:
Removed this part of the patch. I am still not sure how harfbuzz-gobject.dll will work.